internal/cli/serverless/dataimport/start/start.go

```diff
@@ -130,85 +148,121 @@ func StartCmd(h *internal.Helper) *cobra.Command {
 					return err
 				}
 			} else {
-				sourceType = SourceType(cmd.Flag(flag.SourceType).Value.String())
+				sourceType = imp.ImportSourceTypeEnum(cmd.Flag(flag.SourceType).Value.String())
```

Comment:
suggest first input project -> cluster and then import options. Like other commands